simcon_project/templates/template_management/template_delete_modal.html

```diff
@@ -10,8 +10,8 @@ <h3 class="modal-title">Delete Template</h3>
   </div>
 
   <div class="modal-body">
-    <p>Are you sure you want to delete
-      <strong>{{ template.name }}</strong>?</p>
+    <p>Are you sure you want to delete <strong>{{ template_name }}</strong>?</p>
+    <p>All data related to this template will be deleted permanently.</p>
```

Comment:
"All responses and assignments that include this template will be deleted permanently"

Reply:
Wait are the assignments suppose to be deleted too?